libs/internals/core.write_compiled_resource.php

```diff
@@ -6,6 +6,45 @@
  */
 
 /**
+ * Check to see if the path is writable, works on Windows platforms
+ * Based from http://php.net/manual/en/function.is-writable.php comment by
+ * legolas558
+ *
+ * Added 04/03/2015 by Yola to address is_writable PHP bug that does not
+ * correctly handle ACL on Windows platforms.
+ */
+function isWritable($path)
+{
+    $isWin = strtoupper(substr(php_uname('s'), 0, 3)) === 'WIN';
+
+    if($isWin) {
+        if ($path{strlen($path) - 1} == '/') {
+            return isWritable($path.uniqid(mt_rand()) . '.tmp');
+        } elseif (is_dir($path)) {
+            return isWritable($path . '/' . uniqid(mt_rand()) . 'tmp');
```

Comment:
From: http://php.net/manual/en/dir.constants.php

"DIRECTORY_SEPARATOR is not necessarily needed, PHP always converts / to the appropriate character in its file functions.
It is good practice, though."

I think you've mentioned this before.